libr/bin/format/elf/elf.c

```diff
@@ -2181,7 +2181,12 @@ char* Elf_(r_bin_elf_get_arch)(ELFOBJ *bin) {
 		return strdup("kvx");
 	case EM_LOONGARCH:
 		return strdup ("loongarch");
-	default: return strdup ("x86");
+	case EM_386:
+	case EM_X86_64:
+		return strdup("x86");
+	case EM_NONE:
+		return strdup("No machine");
```

Comment:
```suggestion
		return strdup ("null");
```

what do you think about this? this way it will use the "null" plugin to disassemble, analyze, ..

Reply:
That looks fine!